src/api/worker/rest/DefaultEntityRestCache.ts

```diff
@@ -637,6 +637,13 @@ export class DefaultEntityRestCache implements EntityRestCache {
 						containsEventOfType(updatesArray as Readonly<EntityUpdateData[]>, OperationType.CREATE, instanceId)
 					) {
 						// move for mail is handled in create event.
+					} else if (isSameTypeRef(MailTypeRef, typeRef)) {
+						// delete mailDetails if they are available (as we don't send an event for this type)
```

Comment:
is the reason for not sending events for maildetails documented anywhere? I think a link/reference in that comment would help in the future.